hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestCommitsCommand.java

```diff
@@ -76,17 +76,19 @@ public void init() throws IOException {
         "", TimelineLayoutVersion.VERSION_1, "org.apache.hudi.common.model.HoodieAvroPayload");
   }
 
-  private LinkedHashMap<String, Integer[]> generateData() {
+  private LinkedHashMap<String, Integer[]> generateData() throws Exception {
     // generate data and metadata
     LinkedHashMap<String, Integer[]> data = new LinkedHashMap<>();
     data.put("102", new Integer[] {15, 10});
     data.put("101", new Integer[] {20, 10});
     data.put("100", new Integer[] {15, 15});
 
-    data.forEach((key, value) -> {
+    for (Map.Entry<String, Integer[]> entry : data.entrySet()) {
```

Comment:
Why we should do this refactor?

Reply:
@yanghua In `HoodieTestCommitMetadataGenerator#createCommitFileWithMetadata`, I removed the try catch block by declaring the exception hence the callers need to declare it too. The callers' `forEach` force it to be caught therefore changed to for-loop. I'm trying to adopt a preference of declaring Exception over catching them and for-loop over Stream#forEach to avoid noisy try-catch. Please share your thoughts.

Reply:
Ok, I can accept this explanation.